lilliput.go

```diff
@@ -67,6 +73,29 @@ func isMP4(maybeMP4 []byte) bool {
 	return bytes.HasPrefix(magic, mp42Magic) || bytes.HasPrefix(magic, mp4IsomMagic)
 }
 
+// DeanimateAPNG removes animation chunks from []bytes possibly containing a PNG
+func DeanimateAPNG(maybeApng []byte) []byte {
```

Comment:
`maybeApng` is really confusing here - does this mean "maybe a png", or "maybe animated png"?

Reply:
the latter - the former would be `maybeAPng`